sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusClientBuilderTest.java

```diff
@@ -377,4 +395,8 @@ private static URI getUri(String endpointFormat, String namespace, String domain
                 "Invalid namespace name: %s", namespace), exception);
         }
     }
+
+    public static String getEndPoint() {
```

Comment:
Why a helper method that just returns a property? Replace with TestUtils.getEndpoint();

Reply:
It has been verified that this getter method can be replaced by `TestUtils.getEndPoint()`, now fixed in the new version.